js/scripts/main.js

```diff
@@ -72,7 +72,7 @@ CANVAS.onfocus = () => {
 
 // (3) Set the current ZONE to be the first one we encounter - village.main.
 
-let ZONE = Zone.getZones().village.main;
+let ZONE = Zone.getZones().river.river1;
```

Comment:
This should be village.main if I'm not mistaken.

Reply:
Thanks! I'll change it.